yb-voyager/cmd/importData.go

```diff
@@ -924,12 +924,25 @@ func cleanImportState(state *ImportDataState, tasks []*ImportFileTask) {
 		nonEmptyTableNames := lo.Map(nonEmptyNts, func(nt sqlname.NameTuple, _ int) string {
 			return nt.ForOutput()
 		})
-		utils.PrintAndLog("Non-Empty tables: [%s]", strings.Join(nonEmptyTableNames, ", "))
-		utils.PrintAndLog("The above list of tables on target DB are not empty. ")
-		yes := utils.AskPrompt("Are you sure you want to start afresh without truncating tables")
-		if !yes {
-			utils.ErrExit("Aborting import. Manually truncate the tables on target DB before continuing.")
+		if importerRole == TARGET_DB_IMPORTER_ROLE && truncateTables {
+			// truncate tables only supported for import-data-to-target.
+			utils.PrintAndLog("Truncating non-empty tables on target DB : %v", nonEmptyTableNames)
+			err := tdb.TruncateTables(nonEmptyNts)
+			if err != nil {
+				utils.ErrExit("failed to truncate tables: %s", err)
+			}
+		} else {
+			utils.PrintAndLog("Non-Empty tables: [%s]", strings.Join(nonEmptyTableNames, ", "))
+			utils.PrintAndLog("The above list of tables on target DB are not empty.")
+			if importerRole == TARGET_DB_IMPORTER_ROLE {
+				utils.PrintAndLog("If you wish to truncate them, re-run the import command with --truncate-tables true")
+			}
+			yes := utils.AskPrompt("Do you want to start afresh without truncating tables")
+			if !yes {
+				utils.ErrExit("Aborting import.")
```

Comment:
In case of import data to source repliace, manual truncation is the only option right?
So should we add this - `Manually truncate the tables on target DB before continuing.`